src/components/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -30,142 +32,156 @@ export interface AmbassadorCardProps {
   ambassador: AmbassadorType;
   onClose?: () => void;
   className?: string;
+  disableCardEffects?: boolean;
 }
 
 export default function AmbassadorCard(props: AmbassadorCardProps) {
-  const { ambassadorKey, ambassador, onClose, className } = props;
+  const { ambassadorKey, ambassador, onClose, className, disableCardEffects } =
+    props;
   const images = getAmbassadorImages(ambassadorKey);
   const mod =
     window?.Twitch?.ext?.viewer?.role === "broadcaster" ||
     window?.Twitch?.ext?.viewer?.role === "moderator";
+  const glareOpacity = disableCardEffects ? 0.0 : 0.5;
 
   return (
-    <div
-      className={classes(
-        styles.ambassadorCard,
-        className,
-        ambassador.birth && isBirthday(ambassador.birth) && styles.birthday,
-      )}
+    <Tilt
+      tiltEnable={!disableCardEffects}
+      glareEnable={!disableCardEffects}
+      glareMaxOpacity={glareOpacity}
+      glareBorderRadius="1rem"
+      glarePosition="bottom"
+      scale={1.0}
+      perspective={5000}
+      className={classes(styles.ambassadorCard, className)}
     >
-      <div className={styles.hero}>
-        <img
-          className={styles.img}
-          src={images[0].src}
-          alt={images[0].alt}
-          style={{ objectPosition: offsetPosition(images[0].position) }}
-        />
-
-        <div className={styles.overlay}>
-          {props.onClose && (
-            <button
-              className={styles.close}
-              onClick={onClose}
-              type="button"
-              aria-label="Close"
-            >
-              &times;
-            </button>
-          )}
-
-          <h2 className={styles.name} title={ambassador.name}>
-            {ambassador.name}
-          </h2>
+      <div
+        className={classes(
+          ambassador.birth && isBirthday(ambassador.birth) && styles.birthday,
+        )}
+      >
```

Comment:
It looks like this extra inner div has also broken scrolling on the content?

Reply:
Not sure what happened here, will take a look at this and the birthday hat

Reply:
Inner div removed as we now have the Tilt component div.